frontend/public/components/build-config.tsx

```diff
@@ -285,13 +324,29 @@ export const BuildConfigsList: React.SFC<BuildConfigsListProps> = (props) => {
     }),
     [builds, buildsLoaded, buildsLoadError],
   );
+
+  const buildResource = props.data.map((buildConfig) => {
```

Comment:
I like the idea of changing the data instead of using customData. But can we do this as part of the `useMemo`? And update the naming from `customData` to `data` maybe if we don't use `customData` anymore. Let me know if you want chat on this.